tests/test_rewrite_paragraph.py

```
@@ -0,0 +1,702 @@
+        # Now rewrite the paragraph using fresh hash
+        refs2 = doc.list_paragraphs()
+        ref2 = refs2[0].split("|")[0]
+        doc.rewrite_paragraph(ref2, "The board shall review and approve the annual budget proposal.")
+
+        vis = doc.get_visible_text()
+        assert "approve" in vis
+        assert "proceed with" not in vis
+
+    def test_save_and_reopen(self, rewrite_doc):
+        """Document can be saved and reopened after rewrite."""
+        doc, tmp = rewrite_doc
```

Ruff (0.15.7)

[warning] Unpacked variable `tmp` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)